### PR TITLE
added "continue working on" codespaces to nb web doc

### PR DESCRIPTION
**Pull request by minsa110**

### CODE REVIEW

File: docs/datascience/notebooks-web.md [hunk 1]
```diff
@@ -85,14 +90,24 @@ Since VS Code for the Web runs entirely in your web browser, there are some limi
 
 ## Continue working on Codespaces
 
-As with VS Code Desktop, you can easily continue your work from VS Code for the Web ([https://vscode.dev](https://vscode.dev) or [https://github.dev](https://github.dev)) on Codespaces. To do so:
+As with VS Code Desktop, you can easily continue your work from VS Code for the Web ([https://vscode.dev](https://vscode.dev) or [https://github.dev](https://github.dev)) on Codespaces. When you choose to **Continue Working On** Codespaces, your uncommitted changes will travel with you. To do so:
```
Comment (joyceerhl) [inline docs/datascience/notebooks-web.md:93]:
```suggestion
As with VS Code Desktop, you can easily continue your work from VS Code for the Web ([https://vscode.dev](https://vscode.dev) or [https://github.dev](https://github.dev)) on Codespaces. When you choose to **Continue Working On** in a new codespace, your uncommitted changes will travel with you. To do so:
```

File: docs/datascience/notebooks-web.md [hunk 2]
```diff
@@ -15,9 +15,14 @@ Visual Studio Code supports working with [Jupyter Notebooks](https://jupyter-not
...
-- [Remote tunneling with **VS Code Server** on VS Code for the Web](#remote-tunneling-with-vs-code-server)
-- [Connect to a remote **Jupyter server** on VS Code for the Web](#connect-to-a-remote-jupyter-server)
+- [Jupyter Notebooks on the web](#jupyter-notebooks-on-the-web)
+  - [Jupyter notebooks on Codespaces](#jupyter-notebooks-on-codespaces)
+  - [Jupyter notebooks on VS Code for the Web](#jupyter-notebooks-on-vs-code-for-the-web)
+  - [Remote tunneling with VS Code Server](#remote-tunneling-with-vs-code-server)
+  - [Connect to a remote Jupyter server](#connect-to-a-remote-jupyter-server)
+    - [Limitations](#limitations)
+  - [Continue working on Codespaces](#continue-working-on-codespaces)
+  - [Questions or feedback](#questions-or-feedback)
```
Comment (gregvanl) [inline docs/datascience/notebooks-web.md:25]: I would stick with a curated link of sections, if you think it is helpful for readers, but not include the topic hierarchy.

Comment (minsa110) [reply docs/datascience/notebooks-web.md:25]: Changed back. [This extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) was trying to automatically create a "toc" for me..

**Review by gregvanl — APPROVED** [review]
Thanks!

**Pull request merged**